pkg/services/serviceaccounts/database/database.go

```diff
@@ -415,6 +415,11 @@ func (s *ServiceAccountsStoreImpl) SearchOrgServiceAccounts(
 	return searchResult, nil
 }
 
+// AddServiceAccountToTeam adds a service account to a team
+func (s *ServiceAccountsStoreImpl) AddServiceAccountToTeam(ctx context.Context, serviceAccountID int64, cmd *serviceaccounts.AddServiceAccountToTeamCommand) error {
+	return s.sqlStore.AddTeamMember(serviceAccountID, cmd.OrgId, cmd.TeamId, false, models.PERMISSION_ADMIN)
```

Comment:
```suggestion
	return s.sqlStore.AddTeamMember(serviceAccountID, cmd.OrgId, cmd.TeamId, false, models.PERMISSION_ADMIN)
```

How are permissions passed down to this call?

---

Also; should we instead have a implementation of adding a service account to a team outside of `sqlStore` and make it part of the `serviceaccounts.Store` package instead? 🤔

Reply:
We should specify with what permissions we add team member. I guess by default (if we don't want to let SA control team) it should be Viewer.